public/main.js

```diff
@@ -42,103 +53,190 @@ function getTodoList(callback) {
     createRequest.send();
 }
 
+function createButton(id, icon) {
+    var Button = document.createElement("button");
+    Button.setAttribute("type", "button");
+    Button.setAttribute("class", "btn btn-default");
+    Button.setAttribute("id", id);
+    var del = document.createElement("span");
+    del.setAttribute("class", icon);
+    del.setAttribute("aria-hidden", "true");
+    Button.appendChild(del);
+    return Button;
+}
+
+function createReq(method, url, body, errorMsg, callback){
+    fetch( url, {
+        method: method,
+        headers: {
+            "Content-type": "application/json"
+        },
+        body: body
+    })
+    . then(function(res) {
+        console.log("then : " +res);
+        error.textContent = res.responseText;
+        callback();
+    })
+    .catch(function(res){
+        console.log("catch : " +res);
+        error.textContent = errorMsg + " Server returned " + res.status + " - " + res.responseText;
+    });
+}
 
 
 function reloadTodoList() {
     while (todoList.firstChild) {
         todoList.removeChild(todoList.firstChild);
     }
     todoListPlaceholder.style.display = "block";
+
+    var totalItems = 0;
+    var leftItems = 0;
+
+
     getTodoList(function(todos) {
         todoListPlaceholder.style.display = "none";
+
+        var activeTab = document.getElementsByClassName("active")[0].innerText;
+        
+
+        var deleteAllButton = document.createElement("button");
```

Comment:
I see below that you are using the createButton specifically for the buttons with Icons. Its fine to not do this change since this is difference, but at the very least I would rename createButton to make that clear.